bootstrap/kubeadm/config/crd/bases/bootstrap.cluster.x-k8s.io_kubeadmconfigs.yaml

```diff
@@ -1000,6 +1000,7 @@ spec:
                         can be useful (see .node.status.conditions), the ability to deconflict is important.
                       type: string
                   required:
+                  - lastTransitionTime
```

Comment:
Let's talk about this with Fabrizio. I think this happens because CT now supports the upstream +required marker. That was somehow used on v1alpha3 & v1alpha4 (with v1beta1 we dropped omitempty on this field so it became required per default)

Just have to decide what we want to do. Shouldn't matter much as these types are now only used for conversion and not served anymore

Reply:
cc @fabriziopandini

Reply:
xref: https://github.com/kubernetes-sigs/controller-tools/pull/944

Reply:
If this is only making explicit in the OpenAPI spec the fact that this field is required (it is not a pointer, it doesn't have omitempty), then I'm ok with this change

Reply:
I addressed this in the new PR by setting // +optional to keep the CRD YAML identical